src/lib/features/feature-toggle/feature-toggle-strategies-store.ts

```diff
@@ -670,14 +670,24 @@ class FeatureStrategiesStore implements IFeatureStrategiesStore {
             ];
         }
 
-        query = query.select(selectColumns);
+        if (cursor) {
```

Comment:
moved where and order by to match logical order of SQL queries and exclude them from the total count